general/1_terminal/Dima_Mayakov/Solution_task_3.txt

```diff
@@ -0,0 +1,9 @@
+!/bin/bash
+var1=$(date +%u)
+var2=$(date +%d/%b)
```

Comment:
The task is correct, however, let's use more descriptive names than `var1` and `var2`.

Those, after all, have more meaning: they are days.